docs/microfrontend-routing.md

```diff
@@ -0,0 +1,262 @@
+<!-- meta
+{
+  "node": {
+    "label": "Micro frontend routing",
+    "category": {
+      "label": "Advanced",
+      "collapsible": true
+    },
+    "metaData": {
+      "categoryPosition": 7,
+      "position": 2
+    }
+  }
+}
+meta -->
+
+# Micro frontend routing
+
+Luigi routing can be divided in two main parts: higher-level routing of the main app (Luigi Core), and routing within the micro frontend (Luigi Client).
+
+This document shows how you can handle the second way of routing while preserving the existing router of your micro frontend. The exact way this is achieved depends on the type of routing you use, but you should follow some general steps:
+
+1. Enable [virtualTree](navigation-parameters-reference.md#virtualtree) in your Luigi configuration file for the node of your micro frontend.
+2. Import [Luigi Client](luigi-client-setup.md) to your micro frontend.
+3. Use the methods provided by Luigi Client's [linkManager](luigi-client-api.md#linkmanager) to update the main app's URL and maintain navigation history.
+
+Below, you can find some simple examples on how to keep routing in sync for different frontend technologies such as [Angular](#angular-routing), [SvelteKit](#sveltekit-routing), and [React](#react-routing).
+
+<!-- accordion:start -->
+
+## Angular routing
```

Comment:
Here we should add a link to client-support-angular package with the info, that this should be used for the angular use case. Still we should keep the manual setup here for explanation, how it works under the hood.